app/models/course/lesson_plan/item.rb

```diff
@@ -250,9 +250,4 @@ def validate_presence_of_bonus_end_at
     return unless time_bonus_exp && time_bonus_exp > 0 && bonus_end_at.blank?
     errors.add(:bonus_end_at, :required)
   end
-
-  def validate_start_at_cannot_be_after_end_at
```

Comment:
Need to remove https://github.com/Coursemology/coursemology2/pull/3279/files#diff-37c6e834cebb21c2fb93e3e63799cb4eR25 also